webapp/app/statisticalFactsheets/components/table/table.js

```diff
@@ -16,32 +18,43 @@ const Table = (props) => {
 
   const t = (value) => (Number.isNaN(+value) ? i18n.t(`statisticalFactsheets.${section}.${value}`) : value)
   return (
-    <table className="fra-table">
-      <thead>
-        <tr>
-          {columns.map((key) => (
-            <th key={key} className="fra-table__header-cell">
-              {i18n.t(t(key))}
-            </th>
-          ))}
-        </tr>
-      </thead>
-      <tbody>
-        {rows.map((tableRow) => {
-          const row = data.find((entry) => entry.rowName === tableRow)
-          if (!row) return null
-          return (
-            <tr key={row.rowName}>
+    <div className="fra-table__container">
+      <div className="fra-table__scroll-wrapper">
+        <ButtonTableExport tableRef={tableRef} filename={`${section}-${levelIso}`} />
+        <table ref={tableRef} className="fra-table">
+          <thead>
+            <tr>
               {columns.map((key) => (
-                <td key={`${row.rowName}-${row[key]}-${key}`} className="fra-table__calculated-cell">
-                  {t(row[key])}
-                </td>
+                <th key={key} className="fra-table__header-cell">
+                  {i18n.t(t(key))}
+                </th>
               ))}
             </tr>
-          )
-        })}
-      </tbody>
-    </table>
+          </thead>
+          <tbody>
+            {rows.map((tableRow) => {
+              const row = data.find((entry) => entry.rowName === tableRow)
+              if (!row) return null
+              return (
+                <tr key={row.rowName}>
+                  {columns.map((key, i) =>
+                    i === 0 ? (
+                      <th key={`${row.rowName}-${row[key]}-${key}`} className="fra-table__category-cell">
+                        {t(row[key])}
+                      </th>
+                    ) : (
+                      <td key={`${row.rowName}-${row[key]}-${key}`} className="fra-table__calculated-cell">
+                        {t(row[key])}
+                      </td>
+                    )
```

Comment:
if the difference is only the class name, add a conditional class name only

Reply:
td vs th